.github/workflows/rustsec.yml

```diff
@@ -0,0 +1,20 @@
+name: RustSec Audit Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
I would do a security check any time the cargo.toml or cargo.lock files change.

```yaml
on:
  push:
    paths:
      - '**/Cargo.toml'
      - '**/Cargo.lock'
  ```

Reply:
good idea